src/phongo_compat.h

```diff
@@ -152,6 +153,13 @@
 	} else {                                            \
 		ZVAL_LONG((_zv), (_value));                     \
 	}
+#define RETURN_INT64(_value)                               \
```

Comment:
Adding a new compatibility macro felt nicer than using `ZVAL_INT64` on `return_value`. This also made me realise that we don't test any non-Windows 32-bit platforms, but I don't know if we even have 32-bit platforms on evergreen anymore. I'm checking with the build team regarding this.